Resources/Locale/en-US/language/languages.ftl

```diff
@@ -24,3 +24,48 @@ language-Canilunzt-description = The guttural language spoken and utilized by th
 
 language-Moffic-name = Moffic
 language-Moffic-description = The language of the mothpeople borders on complete unintelligibility.
+
+language-RobotTalk-name = RobotTalk
+language-RobotTalk-description = Beep-Boop
```

Comment:
Probably a better description for specifically this one :P

Reply:
Feel free to propose a suggestion for it, i dont have heavly an idea.

Reply:
> An incomprehensible jargon of beeps, numbers, and banging noises. 

Perhaps also change the name to 'Machine' instead of robot.

Reply:
> Feel free to propose a suggestion for it, i dont have heavly an idea.

"A language consisting of harsh binary chirps, whistles, hisses, and whines. Organic tongues cannot speak it without aid from special translators."